apps/studio/components/interfaces/Billing/InvoiceStatusBadge.tsx

```diff
@@ -63,13 +64,22 @@ const InvoiceStatusBadge = ({ status }: InvoiceStatusBadgeProps) => {
               'w-[300px] space-y-2 border border-background',
             ].join(' ')}
           >
-            {[InvoiceStatus.OPEN, InvoiceStatus.UNCOLLECTIBLE].includes(status) && (
-              <p className="text-xs text-foreground">
-                We were not able to collect the money. Make sure you have a valid payment method and
-                enough funds. Outstanding invoices may cause restrictions. You can manually pay the
-                using the "Pay Now" button.
-              </p>
-            )}
+            {[InvoiceStatus.OPEN, InvoiceStatus.ISSUED, InvoiceStatus.UNCOLLECTIBLE].includes(
+              status
+            ) &&
+              (paymentAttempted ? (
+                <p className="text-xs text-foreground">
+                  We were not able to collect the money. Make sure you have a valid payment method
```

Comment:
```suggestion
                  We were not able to collect the payment. Make sure you have a valid payment method
```

money sounds a bit off here, let's align it with your other text, payment also sounds more professional